lib/widgets/leave_request_list.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:cobe_hive_mobile_app/providers/leave_request_provider.dart';
+import 'package:cobe_hive_mobile_app/widgets/leave_request_card.dart';
+import 'package:flutter/material.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+
+class LeaveRequestList extends ConsumerWidget {
+  const LeaveRequestList({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final leaveRequestList = ref.watch(leaveRequestListProvider);
+    return ListView.separated(
+      itemBuilder: (context, index) {
+        final request = leaveRequestList[index];
+        return LeaveRequestCard(leaveRequest: request);
+      },
+      separatorBuilder: (context, index) => const SizedBox(
+        width: 18,
+      ),
+      itemCount: leaveRequestList.length,
+      scrollDirection: Axis.horizontal,
+      padding: const EdgeInsets.symmetric(horizontal: 18),
```

Comment:
separatorBuilder and itemBuilder should be the last parameter in order. itemCount, scroll direction and padding should be at the top